ipam/ipam.go

```diff
@@ -352,5 +332,5 @@ func CreateSchema(rootServiceUrl string, overwrite bool) error {
 	if err != nil {
 		return err
 	}
-	return ipamSvc.store.createSchema(overwrite)
+	return ipamSvc.store.CreateSchema(overwrite)
```

Comment:
No, I mean, I really don't know where "ipamSvc" was defined or what it is, so I'm wondering if maybe you meant to refer to "ipam" instead?